test/fixtures/master.cjs

```
@@ -0,0 +1,52 @@
+  .on('listening', function(worker, address) {
+    console.warn('[%s] [test:worker:%d] listening on %j', Date(), worker.process.pid, address.port);
+    process.send('listening');
+  })
+  .on('disconnect', function(worker) {
+    const propertyName = worker.hasOwnProperty('exitedAfterDisconnect') ? 'exitedAfterDisconnect' : 'suicide';
+    console.warn('[%s] [test:master:%s] worker:%s disconnect, %s: %s, state: %s.',
+      Date(), process.pid, worker.process.pid, propertyName, worker[propertyName], worker.state);
+  })
+  .on('exit', function(worker, code, signal) {
+    const exitCode = worker.process.exitCode;
+    const propertyName = worker.hasOwnProperty('exitedAfterDisconnect') ? 'exitedAfterDisconnect' : 'suicide';
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)